core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -829,6 +881,7 @@ class ZKConfig(props: VerifiableProperties) {
 
 object ZkPath {
   @volatile private var isNamespacePresent: Boolean = false
+  import scala.collection.JavaConversions._
```

Comment:
done, but I couldn't simply import java.util.List. I got a few compilation errors, and I decided to declare java.util.List explicitly in each instance. Hope it is ok.